Bottom: 5622585ae3a1262e6ab63cce9e618161ad1a939d
Top:    ec22d268002ea0e37bbebe94a52c5909a8afa69b
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-31 17:29:11 +0800

Refresh of vhost_net-move-tx-zerocopy

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 31973c8..5091c46 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -1020,12 +1020,12 @@ static void vhost_net_flush(struct vhost_net *n)
 	vhost_net_flush_vq(n, VHOST_NET_VQ_RX);
 	if (nvq->ubufs) {
 		mutex_lock(&vq->mutex);
-		n->tx_flush = true;
+		nvq->tx_flush = true;
 		mutex_unlock(&vq->mutex);
 		/* Wait for all lower device DMAs done. */
 		vhost_net_ubuf_put_and_wait(nvq->ubufs);
 		mutex_lock(&vq->mutex);
-		n->tx_flush = false;
+		nvq->tx_flush = false;
 		atomic_set(&n->vqs[VHOST_NET_VQ_TX].ubufs->refcount, 1);
 		mutex_unlock(&vq->mutex);
 	}
